dns/opnblock/src/opnsense/mvc/app/controllers/OPNsense/opnblock/IndexController.php

```diff
@@ -0,0 +1,12 @@
+<?php
+namespace OPNsense\opnblock;
+class IndexController extends \OPNsense\Base\IndexController
+{
+    public function indexAction()
+    {
+        $this->view->general = $this->getForm("general");
+        $this->view->pick('OPNsense/opnblock/index');
+
+        $this->view->generalForm = $this->getForm("general");
```

Comment:
please don't create redundant variables. You can very likely use the same on all places.